caluma/caluma_form/tests/test_question.py

```diff
@@ -1109,34 +1106,6 @@ def test_recursive_calculated_question(
     assert calc_ans.value == 8
 
 
-def test_calculated_question_update_calc_expr(
-    db, schema_executor, form_and_document, form_question_factory
-):
-    form, document, questions_dict, answers_dict = form_and_document(True, True)
-
-    sub_question = questions_dict["sub_question"]
-    sub_question.type = models.Question.TYPE_INTEGER
-    sub_question.save()
-    sub_question_a = answers_dict["sub_question"]
-    sub_question_a.value = 100
-    sub_question_a.save()
-
-    calc_question = form_question_factory(
-        form=form,
-        question__slug="calc_question",
-        question__type=models.Question.TYPE_CALCULATED_FLOAT,
-        question__calc_expression="'sub_question'|answer + 1",
-    ).question
-
-    calc_ans = document.answers.get(question_id="calc_question")
-    assert calc_ans.value == 101
-
-    calc_question.calc_expression = "'sub_question'|answer -1"
-    calc_question.save()
-    calc_ans.refresh_from_db()
-    assert calc_ans.value == 99
```

Comment:
This functionality was dropped. When updating the calc expression of a calculated question, the answers are not updated automatically anymore.